packages/settings-library/src/settings_library/redis.py

```diff
@@ -18,6 +18,7 @@ class RedisDatabase(IntEnum):
     DISTRIBUTED_IDENTIFIERS = 6
     DEFERRED_TASKS = 7
     DYNAMIC_SERVICES = 8
+    CELERY_TASKS = 9
```

Comment:
This change necesitates an accompanying code change in the ops-tooling repos in the redis-commander.

IMPORTANT: 
- Don't merge this PR as is without the ops-change to come along with it
- Add a "DevOps Warning" in your PR title (look at closed PRs for parenthesis with a construction icon warning as an example).


For more information please:
- Check this https://github.com/ITISFoundation/osparc-ops-environments/blob/main/services/redis-commander/config.json.j2
- Check this https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/deployments/osparc-master-zmt.click/repo.config.template?ref_type=heads#L435
- Talk to me or @YuryHrytsuk for an introduction how this works :--)